akka-http/src/main/scala/akka/http/javadsl/server/directives/WebSocketDirectives.scala

```diff
@@ -44,17 +44,16 @@ abstract class WebSocketDirectives extends SecurityDirectives {
    * Handles WebSocket requests with the given handler and rejects other requests with an
    * [[ExpectedWebSocketRequestRejection]].
    */
-  def handleWebSocketMessages[T](handler: Flow[Message, Message, T]): Route = RouteAdapter {
+  def handleWebSocketMessages(handler: Flow[Message, Message, NotUsed]): Route = RouteAdapter {
```

Comment:
It works actually. However, IMO, I don't think a type parameter is necessary here so I change the signature and make it consistent with other methods in WS (no matter javadsl or scaladsl).